src/azure-cli/azure/cli/command_modules/sql/aaz/latest/sql/mi/link/_create.py

```diff
@@ -64,23 +68,17 @@ def _build_arguments_schema(cls, *args, **kwargs):
         _args_schema.primary_availability_group_name = AAZStrArg(
             options=["--primary-ag", "--primary-availability-group-name"],
             arg_group="Properties",
-            help="Primary availability group name",
-        )
-        _args_schema.replication_mode = AAZStrArg(
-            options=["--replication-mode"],
-            arg_group="Properties",
-            help="Replication mode of a distributed availability group. Parameter will be ignored during link creation.",
-            enum={"Async": "Async", "Sync": "Sync"},
+            help="Name of the primary availability group.",
         )
```

Comment:
Removing `--replication-mode` is a breaking change, please add history notes in PR description following https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#format-pr-description

Reply:
Thanks, updated the description.

Though I'm not 100% sure if this classifies as a breaking change or a bugfix.
`--replication-mode` would've been ignored in `create` command anyways (ignored by the API), and it should only be used with the `update` command.